spec/IndexFolder_Oncocore/genomics.html

```diff
@@ -2,7 +2,7 @@ <h1><a name="genomics"></a>Genomics Reporting</h1>
 
 <p>Cancer genomics is the study of the totality of DNA sequence and gene expression differences between tumor cells and normal host cells. It aims to understand the genetic basis of tumor cell proliferation and the evolution of the cancer genome under mutation and selection by the body environment, the immune system and therapeutic interventions (source: <a href="https://www.nature.com/subjects/cancer-genomics">Nature</a>).</p>
 
-<p>mCODE includes the minimal set of genetic related elements relevant to capture in an EHR to inform cancer assessment and  treatment options, and is loosely based on the <a href="http://build.fhir.org/ig/HL7/genomics-reporting/general.html">HL7 CGWG Clinical Genomics Reporting Implementation Guide</a>. mCODE greatly simplifies genomics reporting to only capture single discrete variants or to variants which were found in a given DNA region.</p>
+<p>mCODE includes the minimal set of genetic related elements relevant to capture in an EHR to inform cancer assessment and treatment options. The approach is based on the <a href="https://www.hl7.org/fhir/genomics.html">HL7 CGWG Clinical Genomics Reporting Implementation Guide</a>, however, mCODE simplifies genomics reporting to only capture single discrete variants or to variants which were found in a given DNA region.</p>
```

Comment:
the official hl7.org/fhir version of the ClinicalGenomics FHIR IG was an older version.  This will hopefully be updated in the September ballot, but if not, then it might not be correct.